packages/xod-client-browser/webpack.config.js

```diff
@@ -5,14 +5,14 @@ const CopyWebpackPlugin = require('copy-webpack-plugin');
 const autoprefixer = require('autoprefixer');
 
 const pkgpath = subpath => path.join(__dirname, subpath);
-const assetsPath = fs.realpathSync(pkgpath('node_modules/xod-client/src/core/assets'));
-const fontAwesomePath = fs.realpathSync(pkgpath('node_modules/xod-client/node_modules/font-awesome'));
+const assetsPath = fs.realpathSync(pkgpath('../../node_modules/xod-client/src/core/assets'));
+const fontAwesomePath = fs.realpathSync(pkgpath('../../node_modules/font-awesome'));
 
 module.exports = {
   devtool: 'source-map',
   entry: [
     'babel-polyfill',
-    pkgpath('node_modules/xod-client/src/core/styles/main.scss'),
+    pkgpath('../../node_modules/xod-client/src/core/styles/main.scss'),
```

Comment:
Looks fragile. Wouldn’t it a better idea to use that Node API function that searches for a package in parent directories? Something like:

```js
// modpath
//  :: String -- module name
//  -> String -- module path
//  -> String -- absolute path

...

entry: [
  'babel-polyfill',
  modpath('xod-client', 'src/core/styles/main.scss'),
  // ...
]
```